src/app/shared/guide-items/guide-items.ts

```diff
@@ -64,9 +64,15 @@ const GUIDES = [
   },
   {
     id: 'using-component-harnesses',
-    name: `Testing with component harnesses`,
+    name: 'Testing with component harnesses',
     document: '/docs-content/guides/using-component-harnesses.html',
     overview: 'Write tests with component harnesses for convenience and meaningful results.'
+  },
+  {
+    id: 'duplicate-theming-styles',
+    name: 'Duplicate theming styles',
+    document: '/docs-content/guides/duplicate-theming-styles.html',
+    overview: 'Read about how to avoidi duplicated theming styles.'
```

Comment:
```suggestion
    overview: 'Learn about our new color mixins for Sass that avoid duplicating theming styles.'
```
Fixed "avoidi" typo and added some context.